Nodejs/Product/Nodejs/TestFrameworks/mocha/mocha.js

```diff
@@ -162,29 +178,21 @@ var run_tests = function (testCases, callback) {
         post({
             type: 'result',
             title: result.title,
-            result: result
+            result: result,
+            pending: false
```

Comment:
You don't need to report the 'pending' property here, it is unused. It is only used from the 'result' object.